LICENSE

```diff
@@ -1,7 +1,15 @@
-Copyright (c) <year> <copyright-holders>
+Copyright 2023 IMAGINARY gGmbH. All Rights Reserved.
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
I don't think it is necessary or even correct to assign the copyright to Google. I am not reusing or modifying any code from Google's Magenta project. PianoGenie is included as a library through NPM.